app/src/main/java/dev/aldi/sayuti/editor/manage/ManageLocalLibraryActivity.java

```diff
@@ -207,21 +208,43 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
             ((ImageView) convertView.findViewById(2131231132)).setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    PopupMenu popupMenu = new PopupMenu(ManageLocalLibraryActivity.this, v);
-                    popupMenu.getMenu().add(0, 0, 0, "Delete");
-                    popupMenu.setOnMenuItemClickListener(new PopupMenu.OnMenuItemClickListener() {
-                        @Override
-                        public boolean onMenuItemClick(MenuItem menuItem) {
-                            FileUtil.deleteFile(local_libs_path.concat(checkBox.getText().toString()));
-                            bB.a(ManageLocalLibraryActivity.this, "Deleted successfully", 0).show();
-                            loadFiles();
-                            return true;
-                        }
-                    });
-                    popupMenu.show();
+                    @Override
+    public boolean onMenuItemClick(MenuItem item){
```

Comment:
Seems like a good idea, but this section is broken. You probably forgot to add something?

Reply:
Actually its because of the text editor. We can fix later after commiting